Source/Core/DolphinQt/Settings/USBDeviceAddToWhitelistDialog.cpp

```diff
@@ -124,8 +124,8 @@ void USBDeviceAddToWhitelistDialog::RefreshDeviceList()
 
 void USBDeviceAddToWhitelistDialog::AddUSBDeviceToWhitelist()
 {
-  const std::string vid_string = StripSpaces(device_vid_textbox->text().toStdString());
-  const std::string pid_string = StripSpaces(device_pid_textbox->text().toStdString());
+  const std::string vid_string(StripSpaces(device_vid_textbox->text().toStdString()));
+  const std::string pid_string(StripSpaces(device_pid_textbox->text().toStdString()));
```

Comment:
Does `IsValidUSBIDString` require null termination? If not, those could be `string_view`.